test/manual-test-examples/webgl/pixels/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="">
+  <script language="javascript" type="text/javascript" src="../../../../lib/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="sketch.js"></script>
+  <style>
+    html, body {margin:0; padding:0;}
+  </style> 
+</head>
+<body>
+<h2 style='margin-left:100px'>If you click the canvas, the box should get textured with an image of the canvas</h2>
```

Comment:
The textured cube is pretty subtle when I view it. It becomes just a tiny speck. Maybe changing the background color at the same time would help. Or using the copied pixels as screengrabs that tile the page.

![screen shot 2017-07-20 at 12 04 57 pm](https://user-images.githubusercontent.com/81553/28412209-b99e9dde-6d43-11e7-9ae6-c83447b47a10.png)
![screen shot 2017-07-20 at 12 05 02 pm](https://user-images.githubusercontent.com/81553/28412208-b99dae88-6d43-11e7-90f1-312c9d929df3.png)